egs/an4/asr1/run.sh

```diff
@@ -245,15 +246,15 @@ if [ ${stage} -le 4 ] && [ ${stop_stage} -ge 4 ]; then
             --penalty ${penalty} \
             --maxlenratio ${maxlenratio} \
             --minlenratio ${minlenratio} \
-            --ctc-weight ${ctc_weight} \
-            &
-        wait
+            --ctc-weight ${ctc_weight}
 
         score_sclite.sh ${expdir}/${decode_dir} ${dict}
 
     ) &
+    pids+=($!)
     done
-    wait
+    for pid in "${pids[@]}"; do wait ${pid} || { echo "Some jobs failed"; exit 1; }; done
```

Comment:
How about `echo "Some background jobs are failed";`
Also, do we need `|| { echo "Some jobs failed"; exit 1; }`?